VideoAPI/VideoApi.IntegrationTests/Database/Commands/SaveHeartbeatCommandTest.cs

```diff
@@ -31,32 +31,32 @@ public async Task should_save_heartbeats()
             _newConferenceId = seededConference.Id;
             var participantId = seededConference.GetParticipants().First().Id;
             
-            var command = new SaveHeartbeatCommand(_newConferenceId, participantId, 1,1,1,1,1,1,1,1, "chrome", "1");
+            var command = new SaveHeartbeatCommand(_newConferenceId, participantId, 1,1,1,1,1,1,1,1, DateTime.UtcNow, "chrome", "1");
             await _handler.Handle(command);
 
-            Heartbeat savedMonitor;
+            Heartbeat savedHeartbeat;
             await using (var db = new VideoApiDbContext(VideoBookingsDbContextOptions))
             {
-                savedMonitor = await db.Heartbeats.FirstOrDefaultAsync(x =>
+                savedHeartbeat = await db.Heartbeats.FirstOrDefaultAsync(x =>
                     x.ConferenceId == _newConferenceId && x.ParticipantId == participantId);
             }
 
-            savedMonitor.Should().NotBeNull();
-            savedMonitor.ConferenceId.Should().Be(_newConferenceId);
-            savedMonitor.ParticipantId.Should().Be(participantId);
-            savedMonitor.OutgoingAudioPercentageLost.Should().Be(1);
-            savedMonitor.OutgoingAudioPercentageLostRecent.Should().Be(1);
-            savedMonitor.IncomingAudioPercentageLost.Should().Be(1);
-            savedMonitor.IncomingAudioPercentageLostRecent.Should().Be(1);
-            savedMonitor.OutgoingVideoPercentageLost.Should().Be(1);
-            savedMonitor.OutgoingVideoPercentageLostRecent.Should().Be(1);
-            savedMonitor.IncomingVideoPercentageLost.Should().Be(1);
-            savedMonitor.IncomingVideoPercentageLostRecent.Should().Be(1);
-            savedMonitor.BrowserName.Should().Be("chrome");
-            savedMonitor.BrowserVersion.Should().Be("1");
-            savedMonitor.Timestamp.Should().NotBe(new DateTime());
-            savedMonitor.Timestamp.Should().BeAfter(DateTime.MinValue);
-            savedMonitor.Timestamp.Should().BeBefore(DateTime.MaxValue);
+            savedHeartbeat.Should().NotBeNull();
```

Comment:
Maybe consider instantiating a new Hearbeat object and then you could do something like 

`savedHearbeat.Should().BeEquivalentTo(expectedHearbeat)`

instead of asserting each properting per line

Reply:
Good point but I am asserting properties to be explicit. But any way would work